runtime/j9vm/jvm.c

```diff
@@ -4412,7 +4414,7 @@ JVM_NativePath(char* path)
 }
 
 /**
- * Checks if a signal is special.
+ * Checks if a signal is special (used for shutdown).
```

Comment:
Please rename the function to better correspond to what is does.

Reply:
SIGHUP, SIGINT and SIGTERM are used for shutdown. so, renamed function to `isSignalUsedForShutdown`.